dockerhooks/dockerhooks.go

```diff
@@ -13,9 +13,6 @@ import (
 
 func getHooks() (string, []os.FileInfo, error) {
 	hookPath := os.Getenv("DOCKER_HOOKS_PATH")
-	if hookPath != "" {
```

Comment:
Yes I fixed this and updated the patch.  Sorry about not being able to fix.  I also found a couple of other problems which I fixed.  

Please take one last look and we will submit upstream on Monday.